README.md

```diff
@@ -1,20 +0,0 @@
-> [!CAUTION]
-> 
-> The docs in the `docs` folder have been moved to the centralized ScalarDB documentation repository, [docs-internal-scalardb](https://github.com/scalar-labs/docs-internal-scalardb). Please update this documentation in that repository instead.
-> 
-> To view the ScalarDB documentation, visit [ScalarDB Documentation](https://scalardb.scalar-labs.com/docs/).
-
-# ScalarDB Samples
-
-This repository contains sample applications for [ScalarDB](https://github.com/scalar-labs/scalardb):
-
-- [ScalarDB Sample](scalardb-sample/)
-- [ScalarDB Server Sample](scalardb-server-sample/)
-- [Multi-storage Transaction Sample](multi-storage-transaction-sample/)
-- [Microservice Transaction Sample](microservice-transaction-sample/)
-- [ScalarDB GraphQL Sample](scalardb-graphql-sample/)
-- [ScalarDB SQL (JDBC) Sample](scalardb-sql-jdbc-sample/)
-- [Spring Data JDBC for ScalarDB Sample](spring-data-sample/)
-- [Spring Data JDBC for ScalarDB with Multi-storage Transaction Sample](spring-data-multi-storage-transaction-sample/)
-- [Spring Data JDBC for ScalarDB with Microservice Transaction Sample](spring-data-microservice-transaction-sample/)
-- [ScalarDB Analytics with PostgreSQL](scalardb-analytics-postgresql-sample/)
```

Comment:
I think we should keep the root README so that we can describe the purpose of this repository and direct users to the docs site. My reasoning behind this is that, some users who find this repository might not know that we have a docs site that provides tutorials on how to run these samples. 

What do you think about changing this as follows?:

```suggestion
# ScalarDB Samples

This repository contains sample applications for [ScalarDB](https://github.com/scalar-labs/scalardb).

To view the ScalarDB documentation, visit [ScalarDB Documentation](https://scalardb.scalar-labs.com/docs/)
```

Reply:
Thank you for your suggestion! I've applied your suggestion. Please take a look again when you have time!